### PR TITLE
Fixes Tom description

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Tom now uses his custom description instead of generic mouse one.
Removed icon_state for gray mouse since its already set.
## Why It's Good For The Game

Tom should be using his custom description, not a generic one.

## Testing

- Took a look into other mouses icons and descriptions, all look good.
## Changelog
:cl:
fix: Tom now has the correct description
/:cl:

### CODE REVIEW

**Comment by Contrabang** [comment]
`icon_state` is needed for mapping the mice, as they will show as the default color otherwise. Only one that might not need it is `mouse_gray`

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 1]
```diff
@@ -79,6 +79,9 @@
 	icon_living = "mouse_[mouse_color]"
 	icon_dead = "mouse_[mouse_color]_dead"
 	icon_resting = "mouse_[mouse_color]_sleep"
+	set_mouse_desc()
+
+/mob/living/simple_animal/mouse/proc/set_mouse_desc()
```
Comment (Sirryan2002) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm:84]: is there any particular reason why this has to be abstracted into its own proc?

Comment (Henri215) [reply code/modules/mob/living/simple_animal/friendly/mouse.dm:84]: image
Doing like this is better? I have been suggested to make a proc but maybe it isnt the best option.

**Review by Vi3trice — COMMENTED** [review]
Picking on it a bit

**Review by farie82 — APPROVED** [review]

**Pull request merged**